### PR TITLE
changed no-undef to be on

### PR DESCRIPTION
**Pull request by sfabriece**
I'm new to eslint configuration. Please guide to the correct way to change the no-undef flag.

### CODE REVIEW

**Comment by AdriVanHoudt** [comment]
@cjihrig mentioned to add `{ "typeof": false }` as config of the rule, you can see how this is done on the hapi/hapi-for-you for example
Also you can mention issues in your commit so github will link them both together, just add `closes #45` to the end of your commit ;)

**Comment by AdriVanHoudt** [comment]
👍 the build is failing since you have to use single quotes ;)
I think you also need tests but you can look into the test folder for examples
Oh and it should definitely be 'error' not 2, both are the same but the number is deprecated

File: lib/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
     'no-empty': 'off',
     'no-native-reassign': 'off',
     'no-underscore-dangle': 'off',
-    'no-undef': 'off',
+    'no-undef': ['error', {'typeof': false}],
```
Comment (cjihrig) [inline lib/index.js]: Please add spaces inside of the curly braces.

**Comment by cjihrig** [comment]
@AdriVanHoudt thanks for helping with this.

@sfabriece thanks for the PR. Can you please add a test.

**Comment by AdriVanHoudt** [comment]
@cjihrig all @sfabriece's work ^^

**Comment by cjihrig** [comment]
I know. I meant the help reviewing.

**Comment by sfabriece** [comment]
I can't seem to run test on a fresh clone on windows. 
When I run `npm test` I get a bunch of errors.

```
Problems in: C:\Users\fabriece\Documents\GitHub\Node\eslint-config-hapi\node_modules\es5-ext\test\error\#\throw.js
        File ignored by default. Use "--ignore-pattern '!node_modules/*'" to override at line [undefined], column [undefined] - (undefined)
 
```

**Comment by cjihrig** [comment]
Sorry, I don't have a Windows machine to try with.

**Comment by cjihrig** [comment]
Landed in c0d2550cbacd73bb0cc1ccd2eac4dfce18d5c1a8. I added a test in b1514a98748813606aa0232f0b22a94f97a58eae.

**Pull request closed without merging**